hbase_storage/src/main/java/com/splicemachine/si/impl/SkeletonTxnNetworkLayer.java

```diff
@@ -39,7 +39,8 @@
     @Override
     public void beginTransaction(byte[] rowKey,TxnMessage.TxnInfo txnInfo) throws IOException{
         TxnMessage.TxnLifecycleService service=getLifecycleService(rowKey);
-        ServerRpcController controller=new ServerRpcController();
+        SpliceRpcController controller = new SpliceRpcController();
+        controller.setPriority(HBaseTableDescriptor.HIGH_TABLE_PRIORITY);
```

Comment:
Any reason you used `HBaseTableDescriptor.HIGH_TABLE_PRIORITY` instead of `HConstants.ADMIN_QOS` or `HConstants.HIGH_QOS` ?

Reply:
`getTxn()` originally used `HConstants.HIGH_QOS`. DB-10230 introduced `HBaseTableDescriptor.HIGH_TABLE_PRIORITY/NORMAL_TABLE_PRIORITY` which are mapped to `HConstants.ADMIN_QOS/NORMAL_QOS`. I reused `HBaseTableDescriptor.HIGH_TABLE_PRIORITY` for other transaction RPCs for consistency.